src/_support/Page/Acceptance/Administrator/ContentListPage.php

```diff
@@ -14,71 +14,99 @@
  *
  * @package  Page\Acceptance\Administrator
  *
- * @since    __DEPLOY_VERSION__
+ * @since    4.0.0
  */
 class ContentListPage extends AdminListPage
 {
 	/**
 	 * Link to the article listing page.
 	 *
 	 * @var    string
-	 * @since  __DEPLOY_VERSION__
+	 * @since  4.0.0
 	 */
 	public static $url = "/administrator/index.php?option=com_content&view=articles";
 
 	/**
 	 * Drop Down Toggle Element.
 	 *
-	 * @var    array
-	 * @since  __DEPLOY_VERSION__
+	 * @var    string
+	 * @since  4.0.0
 	 */
-	public static $dropDownToggle = ['xpath' => "//button[contains(@class, 'dropdown-toggle')]"];
+	public static $dropDownToggle =  "//button[contains(@class, 'dropdown-toggle')]";
 
 	/**
 	 * Page object for content body editor field.
 	 *
-	 * @var    array
-	 * @since  __DEPLOY_VERSION__
+	 * @var    string
+	 * @since  4.0.0
 	 */
-	public static $content = ['id' => 'jform_articletext'];
+	public static $content = '#jform_articletext';
 
 	/**
 	 * Page object for the toggle button.
 	 *
 	 * @var    string
-	 * @since  __DEPLOY_VERSION__
+	 * @since  4.0.0
 	 */
 	public static $toggleEditor = "Toggle editor";
 
 	/**
-	 * Locator for article's name field
+	 * Locator for article's featured icon
 	 *
-	 * @var    array
-	 * @since  __DEPLOY_VERSION__
+	 * @var    string
+	 * @since  4.0.0
 	 */
-	public static $seeName = ['xpath' => "//table[@id='articleList']//tr[1]//td[4]"];
+	public static $seeFeatured = "//table[@id='articleList']//*//span[@class='icon-featured']";
```

Comment:
use of wildcard in xpath is going to be very costly, try to be absolutely sure about the exact path please.